migrations/1_initial_migration.ts

```diff
@@ -1,3 +1,8 @@
+import { SignHashArtifacts } from 'signhash';
+import { Deployer } from 'truffle';
+
+declare const artifacts: SignHashArtifacts;
```

Comment:
There are two problems with this approach:
1. Not in all contexts these variables are defined (e.g. `seed`)
2. WebStorm does not recognize them (pretty lousy argument, but I heavily rely on this IDE)

Reply:
I wouldn't consider 1) a major problem here since most of the code here has the `artifacts` injected, but if it's bad for development than let's keep it as it is.